src/modules/refresh.js

```diff
@@ -0,0 +1,13 @@
+export default async function refresh() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/T14uVhRrwlVGFtBvKPwB/scores/');
+  const json = await response.json();
+  const convert = json.result;
+  const list = document.querySelector('#score-container');
+  list.innerHTML = '';
+  console.log(convert);
```

Comment:
- [ ] for best javascript practices kindly remove the  `console.log`  from your repo. it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website, kindly remove it 🙏